examples/text-generation/quantization_tools/unify_measurements.py

```diff
@@ -28,8 +28,9 @@ def unify_measurements(
 
     # save all the jsons paths in the given measurement group
     for measurement in measurement_group:
-        measurement_path = find_measurement_path(measurement, measurements_dir_path, scales, groups_size)
-        measurements_paths.append(measurement_path)
+        measurement_path = find_measurement_path(measurement, measurements_dir_path, groups_size)
+        if measurement_path is not None:
+            measurements_paths.append(measurement_path)
```

Comment:
what should we do if the measurement_path is None?

Reply:
It won't add the path and will not iterate over it in next steps. we shouldn't do anything

Reply:
@Yantom1  it seems you didn't change the function find_measurement_path. can you update that on line 9 def find_measurement_path(measurement, measurements_dir_path, scales, group_size): ?

Reply:
seems this needs PR1496